MAINTAINER.md

```diff
@@ -12,6 +12,46 @@
 - Julien Salleyron [@juliens](https://github.com/juliens)
 - Nicolas Mengin [@nmengin](https://github.com/nmengin)
 
+
+## PR review process:
+
+* If needed, a specific maintainer can be required to make a review (according to its knowledge areas)
+* The status `needs-design-review` is only used in complex/heavy/tricky PRs.
+* From `1` to `2`: 1 design LGTM in comment, by a senior maintainer, if needed.
+* From `2` to `3`: 3 LGTM by any maintainer.
+
+We use [PRM](https://github.com/ldez/prm) to manage locally pull requests.
+
+
+## Bots
+
+### [Myrmica Lobicornis](https://github.com/containous/lobicornis/)
+
+**Update and Merge Pull Request**
+
+As maintainer when you add the third LGTM, you must add `status/3-needs-merge`.
```

Comment:
Wording improvement suggestion:

_The maintainer giving the final LGTM must add the `status/3-needs-merge` label to trigger the merge bot._